USERGUIDE.md

```diff
@@ -723,6 +724,17 @@ And the Mini App will be loaded like the following scheme,
 
 ```mscheme.rakuten//miniapp/index.html?param1=value1&param2=value2```
 
+
+<a id="permissions-from-host-app"></a>
+
+### Permissions required from the Host app
+
+Mini app requires following set of Device permissions that needed to be added in the Info.plist,
+
+* [NSLocationAlwaysAndWhenInUseUsageDescription](https://developer.apple.com/documentation/bundleresources/information_property_list/nslocationalwaysandwheninuseusagedescription) - Location permission required by Mini app to track/get the current location of the user
+* [NSCameraUsageDescription](https://developer.apple.com/documentation/bundleresources/information_property_list/nscamerausagedescription) - Camera permission required by Mini app to take pictures
+* [NSMicrophoneUsageDescription](https://developer.apple.com/documentation/bundleresources/information_property_list/nsmicrophoneusagedescription) - Microphone permission required by Mini app to record a video.
```

Comment:
Do you think we could have a matrix there?
| Permission  | Description | Reason  |
|---|---|---|
| [NSLocationAlwaysAndWhenInUseUsageDescription](https://developer.apple.com/documentation/bundleresources/information_property_list/nslocationalwaysandwheninuseusagedescription)  | Location permission  | required to track/get the current location of the user |
| [NSCameraUsageDescription](https://developer.apple.com/documentation/bundleresources/information_property_list/nscamerausagedescription)  | Camera permission  | required to take pictures |
| [NSMicrophoneUsageDescription](https://developer.apple.com/documentation/bundleresources/information_property_list/nsmicrophoneusagedescription)  | Microphone permission  | required to record a video |

Reply:
Done